APM/PHP: Replace instructions with examples for installation from source (#11826)
DataDog/documentation: content/en/tracing/faq/php-tracer-manual-installation.md

#    in this specific example it results to be /etc/php.d
RUN echo "extension=ddtrace.so" >> /etc/php.d/98-ddtrace.ini
RUN echo "datadog.trace.request_init_hook=/opt/datadog/dd-trace-php/bridge/dd_wrap_autoloader.php" >> /etc/php.d/98-ddtrace.ini
```

#### Ubuntu example

Follow the instructions below to build the PHP extension in a Ubuntu container. Paths might vary depending on the PHP installation method, but the `Dockerfile` below should be easily adaptable to any environment.

```
FROM ubuntu:20.04

ENV DEBIAN_FRONTEND=noninteractive

# required dependencies
RUN apt update; \
    apt install -y build-essential git libcurl4-openssl-dev php7.4-cli php7.4-fpm php7.4-dev

# install composer (https://getcomposer.org/download/)
RUN php -r "copy('https://getcomposer.org/installer', 'composer-setup.php');" && \
    php -r "if (hash_file('sha384', 'composer-setup.php') === '756890a4488ce9024fc62c56153228907f1545c228516cbf63f885e036d37e9a59d27d63f46af1d4d07ee0f76181c7d3') { echo 'Installer verified'; } else { echo 'Installer corrupt'; unlink('composer-setup.php'); } echo PHP_EOL;" && \
    php composer-setup.php --install-dir=/usr/bin --filename=composer

ENV DD_TRACE_VERSION 0.65.1

> # clone dd-tr   ace-php repo (https://github.com/DataDog/dd-trace-php)

Reviewer: whitespace?

Author: good catch 😃

### Install from source

> Installation from source is only recommended if the provided installers do not work in a specific platform. For example on arm64 machines.

Reviewer 2: ```suggestion
Datadog only recommends installation from source if the provided installers do not work in your specific platform. For example, on arm64 machines.
```

#### Centos example

Follow the instructions below to build the PHP extension in a Centos container. Paths might vary depending on the PHP installation method, but the `Dockerfile` below should be easily adaptable to any environment.

```

> FROM centos:8

Reviewer 2: I think we already discussed this in slack, but it would be easier to use this page if you break this code example out into steps and then smaller code examples for each step

#    You can find your extension directory running:
#       - for `php` --> `php -i | grep -i extension_dir`
#       - for `php-fpm` --> `php-fpm -i | grep -i extension_dir`
#    in this specific example it results to be /usr/lib64/php/modules/
RUN cp ./tmp/build_extension/modules/ddtrace.so /usr/lib64/php/modules/

# 3) add the following lines to 98-ddtrace.ini in your ini settings directory.
#    You can find your ini settings directory running:
#       - for `php` --> `php -i | grep -i 'Scan this dir for additional .ini files'`
#       - for `php-fpm` --> `php-fpm -i | grep -i 'Scan this dir for additional .ini files'`
#    in this specific example it results to be /etc/php.d
RUN echo "extension=ddtrace.so" >> /etc/php.d/98-ddtrace.ini
RUN echo "datadog.trace.request_init_hook=/opt/datadog/dd-trace-php/bridge/dd_wrap_autoloader.php" >> /etc/php.d/98-ddtrace.ini
```

#### Ubuntu example

Follow the instructions below to build the PHP extension in a Ubuntu container. Paths might vary depending on the PHP installation method, but the `Dockerfile` below should be easily adaptable to any environment.

```

> FROM ubuntu:20.04

Reviewer 2: same code example feedback
